{ 'stand_up_albert': { 'left_arm': [ { 'duration': 1.13,
                                       'name': u'stand_up',
                                       'positions': [ -1.3978,
                                                      -0.0628,
                                                      -1.5391,
                                                      1.6328,
                                                      -0.7193,
                                                      -0.2503,
                                                      -0.0001],
                                       'starttime': 0.0},
                                     { 'duration': 1.0,
                                       'name': u'stand_up_0',
                                       'positions': [ -1.5878,
                                                      1.2968,
                                                      -0.0002,
                                                      -0.0001,
                                                      -0.7176,
                                                      -0.2511,
                                                      0.0001],
                                       'starttime': 1.13},
                                     { 'duration': 1.44,
                                       'name': u'stand_up_1',
                                       'positions': [ -2.0902,
                                                      0.611,
                                                      0.0024,
                                                      -0.0012,
                                                      -1.2463,
                                                      0.0029,
                                                      0.0023],
                                       'starttime': 2.13},
                                     { 'duration': 1.0,
                                       'name': u'stand_up_1_2',
                                       'positions': [ -2.2876,
                                                      0.4934,
                                                      -0.0042,
                                                      -0.0285,
                                                      -1.2588,
                                                      -0.0098,
                                                      0.0],
                                       'starttime': 3.57},
                                     { 'duration': 1.808,
                                       'name': u'stand_up_2',
                                       'positions': [ -0.268,
                                                      -1.6445,
                                                      -0.0204,
                                                      -0.0428,
                                                      -1.2599,
                                                      -0.017,
                                                      0.0001],
                                       'starttime': 5.553},
                                     { 'duration': 1.636,
                                       'name': u'stand_up_3',
                                       'positions': [ -0.3277,
                                                      -2.4852,
                                                      -0.1177,
                                                      -0.2064,
                                                      0.0692,
                                                      -0.0233,
                                                      0.1897],
                                       'starttime': 7.361},
                                     { 'duration': 2.001,
                                       'name': u'stand_up_4',
                                       'positions': [ -1.7973,
                                                      -1.0732,
                                                      -0.1109,
                                                      -0.206,
                                                      0.0616,
                                                      -0.02,
                                                      0.19],
                                       'starttime': 8.997},
                                     { 'duration': 0.953,
                                       'name': u'stand_up_5',
                                       'positions': [ -0.172,
                                                      0.0622,
                                                      -0.1055,
                                                      -0.2186,
                                                      0.0601,
                                                      -0.02,
                                                      0.19],
                                       'starttime': 12.146},
                                     { 'duration': 0.496,
                                       'name': u'stand_up_6',
                                       'positions': [ -0.17,
                                                      0.5627,
                                                      -0.11,
                                                      -0.22,
                                                      0.06,
                                                      -0.02,
                                                      0.19],
                                       'starttime': 13.195}],
                       'left_leg': [ { 'duration': 1.0,
                                       'name': u'stand_up_0',
                                       'positions': [ -0.0154,
                                                      -0.0693,
                                                      -0.3338,
                                                      0.7752,
                                                      0.8307,
                                                      -0.06],
                                       'starttime': 0.0},
                                     { 'duration': 1.0,
                                       'name': u'stand_up_1',
                                       'positions': [ -0.0091,
                                                      -0.0795,
                                                      0.0164,
                                                      0.5652,
                                                      1.5704,
                                                      -0.0599],
                                       'starttime': 4.562},
                                     { 'duration': 1.812,
                                       'name': u'stand_up_2',
                                       'positions': [ -0.0084,
                                                      -0.0708,
                                                      -1.3907,
                                                      2.1593,
                                                      1.5707,
                                                      -0.06],
                                       'starttime': 5.562},
                                     { 'duration': 1.636,
                                       'name': u'stand_up_3',
                                       'positions': [ -0.0233,
                                                      -0.0724,
                                                      -1.416,
                                                      3.1411,
                                                      1.8208,
                                                      -0.0701],
                                       'starttime': 7.3740000000000006},
                                     { 'duration': 0.948,
                                       'name': u'stand_up_5_left',
                                       'positions': [ 0.0,
                                                      -0.2557,
                                                      -1.57,
                                                      3.1,
                                                      1.56,
                                                      -0.2537],
                                       'starttime': 12.156},
                                     { 'duration': 0.897,
                                       'name': u'stand_up_7_left',
                                       'positions': [ 0.0,
                                                      -0.062,
                                                      -0.7369,
                                                      1.1723,
                                                      0.5921,
                                                      -0.0622],
                                       'starttime': 13.213}],
                       'neck': [ { 'duration': 1.0,
                                   'name': u'stand_up_0',
                                   'positions': [0.0001, -0.9375],
                                   'starttime': 0.0}],
                       'right_arm': [ { 'duration': 1.13,
                                        'name': u'stand_up',
                                        'positions': [ 1.3978,
                                                       0.0628,
                                                       1.5391,
                                                       -1.6328,
                                                       0.7193,
                                                       0.2503,
                                                       0.0001],
                                        'starttime': 0.0},
                                      { 'duration': 1.0,
                                        'name': u'stand_up_0',
                                        'positions': [ 1.5878,
                                                       -1.2968,
                                                       0.0002,
                                                       0.0001,
                                                       0.7176,
                                                       0.2511,
                                                       -0.0001],
                                        'starttime': 1.13},
                                      { 'duration': 1.44,
                                        'name': u'stand_up_1',
                                        'positions': [ 2.0902,
                                                       -0.611,
                                                       -0.0024,
                                                       0.0012,
                                                       1.2463,
                                                       -0.0029,
                                                       -0.0023],
                                        'starttime': 2.13},
                                      { 'duration': 1.0,
                                        'name': u'stand_up_1_2',
                                        'positions': [ 2.2876,
                                                       -0.4934,
                                                       0.0042,
                                                       0.0285,
                                                       1.2588,
                                                       0.0098,
                                                       -0.0],
                                        'starttime': 3.57},
                                      { 'duration': 1.808,
                                        'name': u'stand_up_2',
                                        'positions': [ 0.268,
                                                       1.6445,
                                                       0.0204,
                                                       0.0428,
                                                       1.2599,
                                                       0.017,
                                                       -0.0001],
                                        'starttime': 5.553},
                                      { 'duration': 1.636,
                                        'name': u'stand_up_3',
                                        'positions': [ 0.3277,
                                                       2.4852,
                                                       0.1177,
                                                       0.2064,
                                                       -0.0692,
                                                       0.0233,
                                                       -0.1897],
                                        'starttime': 7.361},
                                      { 'duration': 2.001,
                                        'name': u'stand_up_4',
                                        'positions': [ 1.7973,
                                                       1.0732,
                                                       0.1109,
                                                       0.206,
                                                       -0.0616,
                                                       0.02,
                                                       -0.19],
                                        'starttime': 8.997},
                                      { 'duration': 0.953,
                                        'name': u'stand_up_5',
                                        'positions': [ 0.172,
                                                       -0.0622,
                                                       0.1055,
                                                       0.2186,
                                                       -0.0601,
                                                       0.02,
                                                       -0.19],
                                        'starttime': 12.146},
                                      { 'duration': 0.496,
                                        'name': u'stand_up_6',
                                        'positions': [ 0.17,
                                                       -0.5627,
                                                       0.11,
                                                       0.22,
                                                       -0.06,
                                                       0.02,
                                                       -0.19],
                                        'starttime': 13.181}],
                       'right_leg': [ { 'duration': 1.0,
                                        'name': u'stand_up_0',
                                        'positions': [ 0.0154,
                                                       0.0693,
                                                       0.3338,
                                                       -0.7752,
                                                       -0.8307,
                                                       0.06],
                                        'starttime': 0.0},
                                      { 'duration': 1.0,
                                        'name': u'stand_up_1',
                                        'positions': [ 0.0091,
                                                       0.0795,
                                                       -0.0164,
                                                       -0.5652,
                                                       -1.5704,
                                                       0.0599],
                                        'starttime': 4.53},
                                      { 'duration': 1.812,
                                        'name': u'stand_up_2',
                                        'positions': [ 0.0084,
                                                       0.0708,
                                                       1.3907,
                                                       -2.1593,
                                                       -1.5707,
                                                       0.06],
                                        'starttime': 5.53},
                                      { 'duration': 1.636,
                                        'name': u'stand_up_3',
                                        'positions': [ 0.0233,
                                                       0.0724,
                                                       1.416,
                                                       -3.1411,
                                                       -1.8208,
                                                       0.0701],
                                        'starttime': 7.3420000000000005},
                                      { 'duration': 0.9489999999999998,
                                        'name': u'stand_up_5_right',
                                        'positions': [ -0.0,
                                                       0.2556,
                                                       1.57,
                                                       -3.1,
                                                       -1.57,
                                                       0.2478],
                                        'starttime': 12.159},
                                      { 'duration': 0.899,
                                        'name': u'stand_up_7_right',
                                        'positions': [ -0.0,
                                                       0.0618,
                                                       0.7369,
                                                       -1.1723,
                                                       -0.5921,
                                                       0.0616],
                                        'starttime': 13.176}],
                       'torso': [ { 'duration': 1.0,
                                    'name': u'stand_up_0',
                                    'positions': [0.0001, 0.0107],
                                    'starttime': 0.0},
                                  { 'duration': 1.84,
                                    'name': u'stand_up_2',
                                    'positions': [0.0016, 0.532],
                                    'starttime': 5.541},
                                  { 'duration': 1.636,
                                    'name': u'stand_up_3',
                                    'positions': [-0.0017, 1.5741],
                                    'starttime': 7.381},
                                  { 'duration': 3.131,
                                    'name': u'stand_up_4',
                                    'positions': [-0.007, -0.0831],
                                    'starttime': 9.017}]}}
